src/fsharp/fsiAnyCpu/FsiAnyCPU.fsproj

```diff
@@ -11,10 +11,9 @@
     <ProjectGuid>{8b3e283d-b5fe-4055-9d80-7e3a32f3967b}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <NoWarn>$(NoWarn);62</NoWarn>
-    <AssemblyName>FsiAnyCPU</AssemblyName>
+    <AssemblyName>fsiAnyCPU</AssemblyName>
```

Comment:
This is not high priority. I'm trying to align these to those used by F# on Linux to help prevent these awkward capitalization errors in the longer term.  

 github.com/fsharp/fsharp has always used ``fsiAnyCpu``.  On Linux we'd expect the executable names are ``lowercase.exe``.there is an argument it should have used ``fsianycpu`` , but I think just aligning to the existing F#-on-Linux norm is simplest.